### PR TITLE
TypeError fixes for sentry

### PR DESCRIPTION
**Pull request by milindk8**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes - https://github.com/devtron-labs/devtron/issues/1809

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Manifest.component.tsx [hunk 1]
```diff
@@ -95,6 +95,7 @@ function ManifestComponent({ selectedTab, isDeleted }) {
                 });
         } catch (err) {
             console.log('err', err);
```
[the remark this answers is not in the document]
Comment (milindk8) [reply src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Manifest.component.tsx:97]: Console log is just for reference & showerror is used inside promise to show only errors related to API

**Review by shivani170 — APPROVED** [review]

File: src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Events.component.tsx [hunk 2]
```diff
@@ -25,18 +26,22 @@ function EventsComponent({ selectedTab, isDeleted }) {
...
                 setLoading(false);
             })
             .catch((err) => {
-                console.log('err', err);
+                showError(err)
                 setEvents([]);
                 setLoading(false);
             });
+        } catch (err) {
+            console.log('err', err);
```
Comment (vivek-devtron) [inline src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Events.component.tsx:41]: we should remove all the console.logs as these won't be beneficial for users

File: src/components/v2/appDetails/sourceInfo/environmentStatus/AppStatusDetailModal.tsx [hunk 3]
```diff
@@ -35,7 +35,7 @@ function AppStatusDetailModal({ close, appStreamData, showAppStatusMessage }: {
     const [showSeeMore, setShowSeeMore] = useState(true)
 
     useEffect(() => {
-        const stats = appStreamData?.result?.application?.status.operationState.syncResult.resources.reduce(
+        const stats = appStreamData?.result?.application?.status.operationState.syncResult?.resources.reduce(
```
Comment (vivek-devtron) [inline src/components/v2/appDetails/sourceInfo/environmentStatus/AppStatusDetailModal.tsx:38]: Are we sure that status and status.operationState will always be there?

Comment (milindk8) [reply src/components/v2/appDetails/sourceInfo/environmentStatus/AppStatusDetailModal.tsx]: Changes are done

**Pull request merged**